tests/CurrencyTransformer/SpanishCurrencyTransformerTest.php

```diff
@@ -11,14 +11,21 @@ public function setUp()
 
     public function providerItConvertsMoneyAmountToWords()
     {
+        /*
+        $A=[];
+        for($x=100;$x<3500;$x+=100){
+            for($y=0;$y<35;$y++){
+                $A[]=[$x+$y,'MXN', ''];
+            }
+        }
+        return $A;
+        */
         return [
-            [100, 'PLN', 'uno zloty'],
```

Comment:
Was there any reason for removing those tests?

Reply:
```suggestion
            [100, 'PLN', 'un zloty'],
```